data/korath/korath missions.txt

```diff
@@ -257,3 +257,337 @@ mission "Small Scale Delivery"
+	source "Greenwater"
+	destination "Seleptra Nak"
+	landing
+	passengers 66
+	blocked "You need to be able to carry <bunks> additional crew to salvage the ship. Return here when you have sufficient crew capacity."
+
+	to offer
+		has "Efreti: Distress Call 5: done"
+
+
+mission "Efreti: Distress Call 6 Ships"
+	invisible
+	landing
+	source "Greenwater"
+	to offer
+		has "Efreti: Distress Call 5: done"
+	to fail
+		or
+			has "Efreti: Distress Call 7: failed"
+			has "Efreti: Distress Call 7: aborted"
+			has "Efreti: Distress Call 7: done"
+	npc kill
+		system "Seketra"
+		fleet "Small Kor Mereti" 2
+		government "Kor Mereti"
+		personality
+			staying
+
+
+mission "Efreti: Distress Call 7"
+	landing
+	name "Salvage the <EDC-ship>"
+	description "Salvage the <EDC-ship> and return it to <planet> to claim rights to it."
+	source "Seleptra Nak"
+	destination "Karek Fornati"
+	landing
+	to offer
+		has "Efreti: Distress Call 6: done"
+	on offer
+		conversation
+			`Thanks to your engineer's work, the atmosphere on the ship is to human standards, if a bit tropical. Your crew works on repairs and getting familiar with the ship. Its strong hull appears spaceworthy, but your engineer tells you that you have no weapons, shield regeneration or battery. If the Mereti attack, you are a sitting duck.`
+			choice
+				`	(Fly the <EDC-ship> home)`
+					goto runforit
+				`	(Abandon the ship)`
+					goto abandon
+			
+			label abandon
+			action
+				fail
+			`	Your crew abandons the Korath ship to the elements. It's just one more tragic story amid the ruins of their civilization.`
+				flee
+			
+			label runforit
+			`	You manage to bring up the shields, at least initially, using the <ship>. Your crew on the <EDC-ship> take their stations as you prepare to evade Mereti patrols on your way to <destination>.`
+				launch
+
+	on complete
+		fail "Efreti: Distress Call 6 Ships"
+		conversation
+			action
+				# Ship name should match the <EDC-ship> substitution above.
+				give ship "Charm-Shallop (Distressed)" "Esketari"
+			`There are no crowds waiting for you when you land; rather, the ground crews seem astonished to see the ship. It takes you some time to arrange for repairs. By the time you're finished, you see a pavilion has been set up near the <EDC-ship> and small groups of Korath periodically stop by.`
+			choice
+				`	(Check it out.)`
+				`	(I've got places to be.)`
+					decline
+				
+			`	The tent is larger than you expected. At the back there is a group of Korath arranged like a choir, but standing oddly still. A Korath in some kind of robes stands next to a table piled with sweets and cups of some kind of thick drink. As you approach, you realize that the "choir" is made of holograms. You do a quick mental count and guess that these are the Korath that died on the ship; this is probably some type of memorial. The Korath in robes greets you and presses a cup into your hand.`
```

Comment:
In the previous PR, Will made the point that `robe` is usually referred to in the plural (`robes`). to me, it still feels weird given the singular nature of the characterization right before the word (`in some kind of`). I have two solutions; up to you which to pick.

```suggestion
			`	The tent is larger than you expected. At the back there is a group of Korath arranged like a choir, but standing oddly still. A Korath in robes stands next to a table piled with sweets and cups of some kind of thick drink. As you approach, you realize that the "choir" is made of holograms. You do a quick mental count and guess that these are the Korath that died on the ship; this is probably some type of memorial. The Korath in robes greets you and presses a cup into your hand.`
```
OR
```suggestion
			`	The tent is larger than you expected. At the back there is a group of Korath arranged like a choir, but standing oddly still. A Korath in some kind of robe stands next to a table piled with sweets and cups of some kind of thick drink. As you approach, you realize that the "choir" is made of holograms. You do a quick mental count and guess that these are the Korath that died on the ship; this is probably some type of memorial. The Korath in robes greets you and presses a cup into your hand.`
```